geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -234,8 +257,8 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.listen(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        goog.events.listen(feature, app.DrawEventType.PROPERTYMODIFYEND,
+            this.onFeaturePropertyChange_, undefined, this);
```

Comment:
I think my problems are due to the fact that I'm working with only one feature.
During the edition, I should use a temporary feature and when the user has finished to modify it, I should copy the modified properies into the original one. 
Thus I could use the change event.